substrate/frame/bags-list/src/list/mod.rs

```diff
@@ -342,6 +348,33 @@ impl<T: Config<I>, I: 'static> List<T, I> {
 		Ok(())
 	}
 
+	/// Insert a new id into the lowest threshold possible bag in the list.
+	///
+	/// Returns an error if the list already contains `id`.
+	pub(crate) fn insert_force_lowest(id: T::AccountId, score: T::Score) -> Result<(), ListError> {
+		if Self::contains(&id) {
+			return Err(ListError::Duplicate)
+		}
+
+		// insert at the end of the lowest threshold bag.
+		let bag_score = notional_bag_for::<T, I>(Zero::zero());
```

Comment:
may be my understanding is not correct, but you should find the last bag which might be non zero (upper score) and still has some room for another id, right? Why create a new bag with score zero?